app/services/claims/import_register_placement_schools.rb

```diff
@@ -0,0 +1,19 @@
+require "csv"
+
+class Claims::ImportRegisterPlacementSchools
+  include ServicePattern
+
+  def initialize(csv_file_path:)
+    @csv_file_path = csv_file_path
+  end
+
+  attr_reader :csv_file_path
+
+  def call
+    urns = CSV.read(csv_file_path, headers: true).map { |row| row["urn"] }
+
+    updated_count = School.where(urn: urns).update_all(claims_service: true)
+
+    Rails.logger.debug "#{updated_count} schools updated."
```

Comment:
```suggestion
    Rails.logger.info "#{updated_count} schools updated."
```

I think debug will only print when you have the debug flag enabled but I think this information should be printed regardless of the debug flag